File: app/src/main/java/com/olokogini/moriai/ui/main/chat/data/ChatMessageEntity.java
package com.olokogini.moriai.ui.main.chat.data;

import androidx.room.*;

@Entity(tableName = "chat_message")
public class ChatMessageEntity {

    @PrimaryKey(autoGenerate = true)
    public long id;

    public String message;
    public boolean isUser;
    public long timestamp;

    public ChatMessageEntity(String message, boolean isUser, long timestamp) {
        this.message = message;
        this.isUser = isUser;
        this.timestamp = timestamp;
    }

}

File: app/src/main/java/com/olokogini/moriai/ui/main/chat/data/ChatMessageEntity.kt
package com.olokogini.moriai.ui.main.chat.data

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "chat_message")
data class ChatMessageEntity(
    @PrimaryKey(autoGenerate = true)
    val id: Long = 0,

    val message: String,
    val isUser: Boolean,
    val timestamp: Long
)